src/components/widgets/outputs/OutputsCard.vue

```diff
@@ -6,6 +6,43 @@
     layout-path="dashboard.outputs-card"
     menu-breakpoint="lg"
   >
+    <template #menu>
+      <v-menu
+        bottom
+        left
+        offset-y
+        transition="slide-y-transition"
+        :close-on-content-click="false"
+      >
+        <template #activator="{ on, attrs }">
+          <v-btn
+            fab
+            x-small
+            text
+            v-bind="attrs"
+            class="ms-1 my-1"
+            v-on="on"
+          >
+            <v-icon>
+              $cog
+            </v-icon>
+          </v-btn>
```

Comment:
Layout has been simplified

```suggestion
          <v-btn
            icon
            v-bind="attrs"
            v-on="on"
          >
            <v-icon dense>
              $cog
            </v-icon>
          </v-btn>
```

Reply:
that seems resnable